README.md

```diff
@@ -10,32 +10,31 @@ general information about the RF or special events during the RF.
 
 Flight segments need not be unique or complete: a given time during a flight may belong to
 any number of segments or none at all. Segments may overlap (i.e. a segment of kind `circling`
-may include several segments of kind `circle`) although a time window should belong to only
-one segment of a given kind.  
+may include several segments of kind `circle`).
 
-### Primary flight segment kinds
-Data providers are encouraged to use following the segment kinds to describe broad flight phases.
-Reporting and validation tools within this repo will be based on this list. Each segment should
-include only one of the following in its list of kinds.
+Following are names of segments assembled from a range of platforms describing broad sampling strategies.
+Data providers are encouraged to use these names, especially those in the first, where applicable.
+Feel free to add the the list (e.g. with a Github pull request) if needed.
+
+### Common vocabulary - broad sampling strategy
+The following describe broad sampling goals. Many will contain subsets
+e.g. `cloud` segments will likely contain `subcloud layer` and `cloud layer` segments.
 * ground
+* transit
 * circle
 * cloud
 * calibration
 * profile
-* level
 * axbt
 * rectangle
 * sawtooth
 * racetrack
 * clover
 
-### Secondary flight segment kinds
-Segments may be of arbitrary kind; where the following kind labels are applicable data providers are encouraged
-to use the same kinds across platforms.
-
+### Common vocabulary - subsets/refinements and super-sets
 * circling: describes multiple consecutive circles.
-* profile-upward: provides further detail for `profile` segments
-* profile-downward: provides further detail for `profile` segments
+* profile upward: provides further detail for `profile` segments
+* profile downward: provides further detail for `profile` segments
```

Comment:
I'd suggest to only use `upward` and `downward` as kinds. They could be combined with `profile` and users could filter for `profile` && `upward`, but repeating the word "profile" in these kinds seems to be redundant and unnecessarily specific.
```suggestion
* upward: primarily ascending, could be used as further detail for `profile` segments
* downward: primarily descending, could be used as further detail for `profile` segments
```

Reply:
Agreed with this idea.  I don't see a need to have these be "profiles".  There are definitely times when the aircraft is ascending and descending that I wouldn't count as profiling.